### PR TITLE
Sa/migrate account

### PR DESCRIPTION
**Pull request by chefsalim**
Migrate accounts and invitations to direct DB

tenor-255421298

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-db/src/models/account.rs [hunk 1]
```diff
@@ -1,77 +1,100 @@
...
         diesel::sql_query("SELECT update_account_v1($1, $2)")
-            .bind::<BigInt, _>(account.id)
-            .bind::<Text, _>(account.email)
+            .bind::<BigInt, _>(id as i64)
+            .bind::<Text, _>(email)
+            .execute(conn)
+    }
+}
+
+#[derive(Insertable)]
```
Comment (elliott-davis) [inline components/builder-db/src/models/account.rs]: Do you need to derive insertable here if we're not using the query dsl?

File: components/builder-db/src/models/invitations.rs [hunk 2]
```diff
@@ -25,13 +25,63 @@ pub struct OriginInvitation {
     pub updated_at: Option<NaiveDateTime>,
 }
 
+#[derive(Insertable)]
```
Comment (elliott-davis) [inline components/builder-db/src/models/invitations.rs]: Same question as above

File: components/builder-db/src/models/account.rs (hunk 1 above)
Comment (chefsalim) [reply components/builder-db/src/models/account.rs]: I'm leaving those in mostly as a placeholder for experimenting with direct inserts

**Pull request merged**